unittests/CppInterOp/VariableReflectionTest.cpp

```diff
@@ -121,13 +137,17 @@ TEST(VariableReflectionTest, GetVariableOffset) {
   EXPECT_TRUE((bool) Cpp::GetVariableOffset(Decls[1])); // N
 
   EXPECT_EQ(Cpp::GetVariableOffset(datamembers[0]), 0);
+//FIXME: Unable to compile these on Windows due to type conversion
+#ifdef _WIN32
```

Comment:
What is the error?

Reply:
I'd have to rerun it again without this guard, but I think it was something like  &(c.b),&(c.c) and &(c.d) was coming out as int** type and was unable to convert to unsigned long on Windows. &(c.a) was coming out int* and it was unable to convert to unsigned long.

Reply:
What if we cast to `intptr_t`?

Reply:
I will try this in a few minutes. I have performed a run with your suggestions, and removed this guard so you can see the exact error incase I am misquoting the error.

Reply:
@vgvassilev Here is a link to a run with the error https://github.com/compiler-research/CppInterOp/actions/runs/9142123700/job/25137329640?pr=288#step:16:177 . I will now try your suggestion.

Reply:
> What if we cast to `intptr_t`?

This solution worked.